ros_ign_interfaces/msg/JointWrench.msg

```diff
@@ -0,0 +1,8 @@
+std_msgs/Header header                  # Time stamp
```

Comment:
I know this new message is matching `ignition::msgs::JointWrench`, but I'll just leave a note that we should be modernizing this in the future to use clearer simulation concepts (i.e. `link` or `collision` instead of `body` and `ignition::msgs::Entity` instead of `id` + `name`).

I'd suggest preemptively doing that here now even though it won't match the Ignition message, but I don't have a strong preference.

Reply:
I am OK changing this and also the proto possibly to do this more high level. Do you mind if I do that in another PR and just open an issue to track. I am also down to change any other types as well, I can take care of them in that future PR as well.